Parsers/SysmonParser.txt

```diff
@@ -64,7 +64,6 @@ User = EventDetail.[5].["#text"], Protocol = EventDetail.[6].["#text"], Initiate
 SourceHostname = EventDetail.[10].["#text"], SourcePort = EventDetail.[11].["#text"], SourcePortName = EventDetail.[12].["#text"], DestinationIsIpv6 = EventDetail.[13].["#text"], 
 DestinationIp = EventDetail.[14].["#text"], DestinationHostname = EventDetail.[15].["#text"], DestinationPort = EventDetail.[16].["#text"], DestinationPortName = EventDetail.[17].["#text"]
 | parse RuleName with * 'technique_id=' TechniqueId ',' * 'technique_name=' TechniqueName 
-| parse Hashes with * 'SHA1=' SHA1 ',' * 'MD5=' MD5 ',' * 'SHA256=' SHA256 ',' * 'IMPHASH=' IMPHASH 
 | project-away EventDetail, RuleName, Hashes
```

Comment:
i think you need to lose the "Hashes" in line project away line too, just below

Reply:
removed